README.md

```diff
@@ -1,82 +1,237 @@
-| ClickHouse GRPC Client package | 0.7.0   | Please use the clickhouse http protocol instead  |
-## Usage
-
-The library can be downloaded from both [Github Releases](../../releases) and [Maven Central](https://repo1.maven.org/maven2/com/clickhouse/). Development snapshots(aka. nightly build) are available on [Sonatype OSSRH](https://s01.oss.sonatype.org/content/repositories/snapshots/com/clickhouse/).
-
-```xml
-<repositories>
-    <repository>
-        <id>ossrh</id>
-        <name>Sonatype OSSRH</name>
-        <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
-    </repository>
-</repositories>
+| Clickhouse CLI Client (Java)   | 0.7.0   | Please use `clickhouse-client` (see https://clickhouse.com/docs/en/interfaces/cli#clickhouse-client)                   |
+| ClickHouse GRPC Client         | 0.7.0   | Please use the ClickHouse http client instead. GRPC protos still available https://github.com/ClickHouse/ClickHouse/tree/master/src/Server/grpc_protos   |
+
+
+## Installation
+
+Releases: Maven Central (web site https://mvnrepository.com/artifact/com.clickhouse)
+
+Nightly Builds: https://s01.oss.sonatype.org/content/repositories/snapshots/com/clickhouse/
+
+## Client V2
+
+### Artifacts
+
+| Component                 | Maven Central Link |
+|---------------------------|--------------------|
+| ClickHouse Java Client V2 | [![Maven Central](https://img.shields.io/maven-central/v/com.clickhouse/client-v2)](https://mvnrepository.com/artifact/com.clickhouse/client-v2) |
+
+### Compatibility
+
+| ClickHouse Version | Client Version | Comment                                                                                                                                          |
+|--------------------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| Server >= 23.0     | 0.6.2          |                                                                                                                               |
+
+
+### Features
+
+- Http API for ClickHouse support
+- Bi-directional Compression
+  - LZ4
+- Insert from POJO (data is provided as list of java objects)
+- Query formats support: 
+  - RowBinary readers
+  - Native format reader
+- Apache HTTP Client as HTTP client
+  - Connection pooling
+  - Failures on retry  
+- SSL support
+- Cloud support
+- Proxy support
+
+### Examples
+
+[Begin-with Usage Examples](../../tree/main/examples/client-v2)
+
+[Spring Demo Service](https://github.com/ClickHouse/clickhouse-java/tree/main/examples/demo-service) 
+
+Minimal client setup: 
+```java
+String endpoint = "https://<db-instance hostname>:8443/"
+Client client = new Client.Builder()
+        .addEndpoint(endpoint)
+        .setUsername(user)
+        .setPassword(password)
+        .setDefaultDatabase(database)
+        .build();
+```                
+
+Insert POJOs example:
+```java 
```

Comment:
I don't think we have examples in Readme in other repos, but okay until we have proper docs on the website

Reply:
I put it intentionally because:
- I personally prefer docs closer to code 
- I visit github first and use examples from their
- there always a lag between code repo and doc repo